mosaic/__init__.py

```diff
@@ -315,6 +323,7 @@ def to_hdu_list(self):
                 for key, value in self.mosaic['keywords'].items():
                     logger.debug('Update keyword %s = %s', key, value)
                     h[key] = value
+                h['TELAPSE'] = h['TLAST'] - h['TFIRST']
```

Comment:
TELAPSE should be converted here from days to seconds, no? I am adding tests to cover that. But I thought you;d wanted to improve this right away in your fork.